packages/aragon-cli/package.json

```diff
@@ -8,11 +8,13 @@
     "dao": "./aliases/dao"
   },
   "scripts": {
-    "build": "npm run extract-roles && babel -d dist src --copy-files",
+    "build": "npm run extract-roles && npm run compile:src",
+    "watch": "npm run build -- -- --watch",
```

Comment:
```suggestion
    "start": "npm run build -- -- --watch",
```

Reply:
I think `start` feels kind of weird, since you're not technically starting anything (you'd still need to link it?). What about `build:watch`?

Reply:
Makes sense. Would be awesome if we could come up with a "community standard" like:
`npm start` - for production use (build, link, etc.)
`npm run dev` - for dev use

So you only one command (besides install) to get up and running, without looking in `package.json`.

Would've been cool if the `npm` team "reserved" a script for this. :D